### ISSUE TITLE
EventPath constructor takes only 2 arguments

### ISSUE THREAD

**Issue by jamesonthecrow** [Sentenai/py-sentenai#18]
Currently the EventPath constructor is:

```
def __init__(self, namet=None):
    ...
```

However, the `self._` method attempts to create an EventPath object by passing two arguments:

```
def _(self, name):
        return EventPath(self.__attrlist + (name,), self.__stream)
```

This throws a `TypeError: __init__() takes at most 2 arguments (3 given)`

**Comment by xnomagichash** [comment]
Fixed in 0.3.3.1

**Comment by jamesonthecrow** [comment]
@xnomagichash are you sure this was fixed? The offending code is still in there and I am still getting errors.

https://github.com/Sentenai/py-sentenai/blob/master/sentenai/flare.py#L359
